### DISCUSSION TITLE
why not support  agent dynamic table so that we can't define a base Model

### DISCUSSION THREAD

**Discussion by PuAlbert** [prisma/prisma#18133] [Q&A]
## Problem
why not support  agent dynamic table so that we can't define a base Model

## Suggested solution

First of all, all our database table clients should have a basic stub, not every one is different. The problem caused by this is obvious: dynamic database table structure mapping cannot be done.

**Comment by janpio** [comment]
Prisma is built around generated types for a model, that represents a table. If we do not know the schema of a table, we can not calculate the types.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
